test/auth.test.js

```diff
@@ -126,6 +145,27 @@ describe('Auth linking', () => {
       expect(error).toBeDefined();
     }
   });
+
+  it('should list pending users', async() => {
+    const testEmail = 'test_user@10gen.com';
+    await client.register(testEmail, password);
+    const userId = await client.login();
+    await client.register(linkEmail, password);
+
+    let { token_id: tokenId, token } = await th.app().userRegistrations().sendConfirmationEmail(linkEmail);
```

Comment:
I think you should just use your new `confirmByEmail` endpoint instead of doing the whole sending an email thing. Also you should add a test for when the limit is actually used e.g. more than 3 records the result should only contain 3 records.